packages/cra-template-imodeljs/template/src/api/App.ts

```diff
@@ -0,0 +1,74 @@
+import { ClientRequestContext, Config } from "@bentley/bentleyjs-core";
+import { BrowserAuthorizationCallbackHandler, BrowserAuthorizationClient, BrowserAuthorizationClientConfiguration } from "@bentley/frontend-authorization-client";
+import { BentleyCloudRpcManager, BentleyCloudRpcParams } from "@bentley/imodeljs-common";
+import { FrontendRequestContext, IModelApp } from "@bentley/imodeljs-frontend";
+import { UrlDiscoveryClient } from "@bentley/itwin-client";
+import { UiComponents } from "@bentley/ui-components";
+import { IModelReadRpcInterface, IModelTileRpcInterface, RpcInterfaceDefinition } from "@bentley/imodeljs-common";
+
+/**
+ * Returns a list of RPCs supported by this application
+ */
+export function getSupportedRpcs(): RpcInterfaceDefinition[] {
+  return [
+    IModelReadRpcInterface,
+    IModelTileRpcInterface,
+  ];
+}
+
+// Boiler plate code
+export class iModeljsApp {
+
+  public static get oidcClient() { return IModelApp.authorizationClient as BrowserAuthorizationClient; }
+
+  public static async startup() {
+    await IModelApp.startup({ applicationVersion: "1.0.0" });
+
+    // initialize OIDC
+      await iModeljsApp.initializeOidc();
```

Comment:
nit: there's a few space formatting issues in the file

Reply:
Resolved.